annis-visualizers/src/main/java/annis/visualizers/htmlvis/HTMLVis.java

```diff
@@ -108,7 +119,11 @@ public Panel createComponent(VisualizerInput vi, VisualizationToggle vt)
 
   
     String visConfigName = vi.getMappings().getProperty("config");
-  
+    String hitMarkConfig = vi.getMappings().getProperty("hitmark","on");
```

Comment:
The other visualizers usually use "true" to enable a certain feature (e.g. in http://korpling.github.io/ANNIS/doc/classannis_1_1visualizers_1_1component_1_1grid_1_1_grid_visualizer.html) and I think we should be consistent on this over the different visualizers.